app/qml/components/MMToolbar.qml

```diff
@@ -65,10 +65,19 @@ MMBaseToolbar {
 
   function setupBottomBar() {
     var m = root.model
-    var c = m.count
     var w = buttonView.width
     var button
 
+    for ( var j = m.count - 1 ; j >= 0 ; j-- )
+    {
+      if ( m.get(j).visibilityMode === false )
```

Comment:
You might need to first check if the `visibilityMode` exists. There are buttons like `MMToolbarLongButton` that does not have this new property.

```suggestion
      if ( m.get(j).visibilityMode && m.get(j).visibilityMode === false )
```